include/bluetooth/uuid.h

```diff
@@ -2123,6 +2123,78 @@ struct bt_uuid_128 {
  */
 #define BT_UUID_HAS_ACTIVE_PRESET_INDEX \
 	BT_UUID_DECLARE_16(BT_UUID_HAS_ACTIVE_PRESET_INDEX_VAL)
+/** @def BT_UUID_CGMS_VAL
+ *  @brief Continuous Glucose Monitoring Service UUID value
+ */
+#define BT_UUID_CGMS_VAL 0x181f
+/** @def BT_UUID_CGMS
+ *  @brief Continuous Glucose Monitoring Service
+ */
+#define BT_UUID_CGMS \
+	BT_UUID_DECLARE_16(BT_UUID_CGMS_VAL)
+/** @def BT_UUID_CGM_MEASUREMENT_VAL
+ *  @brief CGM Measurement Characteristic value
+ */
+#define BT_UUID_CGM_MEASUREMENT_VAL 0x2aa7
+/** @def BT_UUID_CGM_MEASUREMENT
+ *  @brief CGM Measurement Characteristic
+ */
```

Comment:
We have so far ordered our UUID entried by increasing UUID value. Please insert these in the correct places based on the values.

Reply:
Updated. They are in ascending order now.